zerver/actions/invites.py

```diff
@@ -431,7 +434,7 @@ def revoke_invites_generated_by_user(user_profile: UserProfile) -> None:
 
     Confirmation.objects.bulk_update(confirmations_to_revoke, ["expiry_date"])
     if len(confirmations_to_revoke):
-        notify_invites_changed(realm=user_profile.realm)
+        notify_invites_changed(realm=user_profile.realm, acting_user=user_profile)
```

Comment:
I think we don't need to send the event here to `user_profile` since this is called as a result of deactivating the user.